packages/grafana-e2e/package.json

```diff
@@ -15,7 +15,14 @@
     "url": "http://github.com/grafana/grafana.git",
     "directory": "packages/grafana-e2e"
   },
-  "main": "src/index.ts",
+  "exports": {
+    ".": {
+      "import": "./src/index.ts",
```

Comment:
Is this for local development? There is no `src` directory in the [bundled e2e package](https://unpkg.com/browse/@grafana/e2e@9.5.2/) which leads me to believe this will cause issues for certain setups. If you need to change this property when bundling to a dist directory the `publishConfig` property in this package.json can help.

Reply:
I have updated so it points to the dist index instead. WDYT?